doc/source/serve/production.md

```diff
@@ -309,41 +309,41 @@ As a side note, you could also package your deployment graph into a standalone P
 By default, `serve deploy` deploys to a cluster running locally. However, you should also use `serve deploy` whenever you want to deploy your Serve application to a remote cluster. `serve deploy` takes in an optional `--address/-a` argument where you can specify the dashboard address of your remote Ray cluster. This address should be of the form:
 
 ```
-[YOUR_RAY_CLUSTER_URI]:[DASHBOARD PORT]
+[YOUR_RAY_CLUSTER_URI]:[DASHBOARD AGENT PORT]
 ```
 
-As an example, the address for the local cluster started by `ray start --head` is `http://127.0.0.1:8265`. We can explicitly deploy to this address using the command
+As an example, the address for the local cluster started by `ray start --head` is `http://127.0.0.1:52365`. We can explicitly deploy to this address using the command
 
 ```console
-$ serve deploy config_file.yaml -a http://127.0.0.1:8265
+$ serve deploy config_file.yaml -a http://127.0.0.1:52365
 ```
 
-The Ray dashboard's default port is 8265. This port may be different if:
-* You explicitly set it using the `--dashboard-port` argument when running `ray start`.
-* Port 8265 was unavailable when Ray started. In that case, the dashboard port is incremented until an available port is found. E.g. if 8265 is unavailable, the port becomes 8266. If that's unavailable, it becomes 8267, and so on.
+The Ray dashboard agent's default port is 52365. This port may be different if:
+* You explicitly set it using the `--dashboard-agent-listen-port` argument when running `ray start`.
+* Port 52365 was unavailable when Ray started. In that case, the dashboard agent port is randomly generated.
```

Comment:
Is this still true? I think now we crash if port is unavailable.

If that's the case, I would remove this bullet point and replace it with a sentence mentioning that if the port was occupied before the agent started, the agent will still run but be unable to respond to Serve CLI requests or REST API requests. I would also mention (1) how the user should check if that's the case (what commands they should run to see if the port is occupied) and (2) what the user should do to make the Serve CLI work again.